docs/introduction.md

```diff
@@ -5,402 +5,103 @@ This document is aimed at introducing developers to the Flow Data Provisioning S
+You should now have the DPS API available at `localhost:5005`.
+It can be used in conjunction with the DPS Rosetta Server and the DPS Access API, which both need the address to your DPS API in order to function.
 
-### Rules
+### Indexing Live Sporks
 
-Flow is based upon the following set of rules:
+The DPS Live binary handles both indexing and serving the DPS API for its index.
+This is because it is impossible for one process to write on the index while another reads from it without causing concurrency issues.
 
-- Consensus and Authentication
-    - All nodes participating in the system are known to each other.
-    - Each node is authenticated through its unforgeable digital signature.
-    - Consensus is based on [Proof of Stake](#proof-of-stake)
-- Participation in Network
-    - The evolution of the chain consists of fixed intervals called epochs
-    - To participate in the network, a node must put up the minimum required [stake](#staking) for that role in a specific epoch.
-    - A node may participate over multiple epochs.
-- Source of randomness
-    - Flow requires a reliable source of randomness for seeding its pseudo-RNG
-    - The source of randomness enables each seed to be unpredictable by any individual node until the seed itself is generated and published in a decentralized manner
-    - This is done using the Distributed Random Beacon (DRB) protocol to generate a fully decentralized, reliable source of randomness.
-- Cryptography Primitives
-    - Flow requires an aggregatable and non-interactive signature scheme, such as [BLS](https://en.wikipedia.org/wiki/BLS_digital_signature)
-- Network Model
-    - Flow operates on a partially synchronous network.
-- Rewarding and Slashing
-    - Flow requires adequate compensation and [slashing](#slashing) mechanics that incentivize nodes to comply with the protocol.
-- Honest Stake Fraction
-    - Flow requires more than two thirds (2/3) of [stake](#staking) from [collection](#collection-nodes), [consensus](#consensus-nodes) and [verification](#verification-nodes) nodes to be controlled by honest actors (for each node role separately). A super-majority (two thirds) of honest nodes probabilistically guarantees the safety of the Flow protocol.
+It needs to connect to the Flow network by acting as an unstaked consensus follower, and needs the following information:
 
+* The spork's root checkpoint
+* The spork's public bootstrap information file
+* The name of the GCS bucket in which block records are available
+* The address of the seed node to follow unstaked consensus
+* The hex-encoded public network key of the seed node to follow unstaked consensus
 
-### Errors
+```console
+$ ./flow-dps-live -u flow-block-data -i /var/flow/index -d /var/flow/data -c /var/flow/bootstrap/root.checkpoint -b /var/flow/bootstrap/public --seed-address access.canary.nodes.onflow.org:9000 --seed-key cfce845fa9b0fb38402640f997233546b10fec3f910bf866c43a0db58ab6a1e4
+```
```

Comment:
I think we can add that this _creates_ the protocol state database at the `-d` location, using the unstaked consensus follower, rather than only reading from it.